net/isc-dhcp/files/dhcpd.init

```diff
@@ -494,9 +528,18 @@ start_service() {
 
 		config_load dhcp
 
+		local rfc1918_nets=""
+
+		# alas we have to make 2 passes...
+		config_foreach dhcpd_add dhcp 0
+
+		rfc1918_nets="$(echo "$rfc1918_nets" | tr ' ' $'\n' | sort | uniq | tr $'\n' ' ')"
```

Comment:
If I had associative arrays in Ash I would have done this "uniquify" by putting using the values as keys into an array and then retrieving the keys.